src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -119,40 +119,27 @@ object Read extends Logging {
     mdTagString: String,
     failedVendorQualityChecks: Boolean = false,
     isPositiveStrand: Boolean = true,
-    matePropertiesOpt: Option[MateProperties] = None): Read = {
+    matePropertiesOpt: Option[MateProperties] = None) = {
 
     val sequenceArray = sequence.map(_.toByte).toArray
     val qualityScoresArray = baseQualityStringToArray(baseQualities, sequenceArray.length)
 
-    if (referenceContig.isEmpty) {
-      UnmappedRead(
```

Comment:
Yes, actually it does, but we were only using it to make `MappedReads` anyways.

`UnmappedRead`'s constructor could be used directly instead.